sqld/src/rpc/replication_log.rs

```diff
@@ -8,23 +8,25 @@ use std::net::SocketAddr;
 use std::pin::Pin;
 use std::sync::{Arc, RwLock};
 
+use bytes::Bytes;
 use futures::stream::BoxStream;
 use tokio::sync::mpsc;
 use tokio_stream::wrappers::ReceiverStream;
 use tokio_stream::StreamExt;
 use tonic::Status;
 
 use crate::auth::Auth;
+use crate::namespace::{NamespaceStore, PrimaryNamespaceMaker};
 use crate::replication::primary::frame_stream::FrameStream;
-use crate::replication::{LogReadError, ReplicationLogger};
+use crate::replication::LogReadError;
 use crate::utils::services::idle_shutdown::IdleShutdownLayer;
 
 use self::rpc::replication_log_server::ReplicationLog;
 use self::rpc::{Frame, Frames, HelloRequest, HelloResponse, LogOffset};
 
 pub struct ReplicationLogService {
-    logger: Arc<ReplicationLogger>,
-    replicas_with_hello: RwLock<HashSet<SocketAddr>>,
+    namespaces: Arc<NamespaceStore<PrimaryNamespaceMaker>>,
+    replicas_with_hello: RwLock<HashSet<(SocketAddr, Bytes)>>,
```

Comment:
Is this bytes supposed to be a namespace? If so I would suggest we maybe newtype this to make sure its clear that this bytes was extracted as a namespace, this can be done in a follow up though.